me.gladwell.eclipse.m2e.android/plugin.xml

```diff
@@ -1,51 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
So we need to go through all the files in the code base and find out which ones are using Windows' delimiters and fix them to Unix?